package.json

```diff
@@ -2202,6 +2202,47 @@
 						"scope": "window",
 						"order": 12
 					},
+					"gitlens.graph.scrollMarkers.enabled": {
+						"type": "boolean",
+						"default": true,
+						"markdownDescription": "Specifies whether to show markers on the scrollbar in the _Commit Graph_",
+						"scope": "window",
+						"order": 13
+					},
+					"gitlens.graph.scrollMarkers.types": {
```

Comment:
Thinking about this more -- I think we take out `selection` and `highlights` from here completely -- and make them always included, unless you turn off the whole feature via `gitlens.graph.scrollMarkers.enabled`

Reply:
I'm ok with that, but as a user I can see myself wanting to, for example, turn `highlights` off and just have `selection` enabled so it's not too noisy/distracting on the right.

Reply:
Done in commit: [Use markers in settings](https://github.com/gitkraken/vscode-gitlens/pull/2489/commits/780016d774cce7e278eb5652ce8499d4bd5dbc51)